.mailmap

```diff
@@ -173,12 +173,12 @@ Imran Iqbal <imran@imraniqbal.org> <imrani@ca.ibm.com>
 Ionică Bizău <bizauionica@gmail.com> <bizauionica@yahoo.com>
 Isaac Z. Schlueter <i@izs.me>
 Isaac Z. Schlueter <i@izs.me> <i@foohack.com>
-Isaac Z. Schlueter <i@izs.me> isaacs <nope@not.real>
+Isaac Z. Schlueter <i@izs.me> <nope@not.real>
```

Comment:
I guess this one should kind of be left as-is, since the identifying part is the author name, not the email?